onebuild/commands/help_command.py

```diff
@@ -0,0 +1,8 @@
+from onebuild.commands.command import Command
+from onebuild.utils import print_help
+
+
+class HelpCommand(Command):
+
+    def execute(self, arg_parser, arguments, build_file_name, command_name):
+        print_help(arg_parser)
```

Comment:
Inline `print_help(arg_parser)` method.

As this is the only use case & `help_command` file seems the perfect place to keep this.

Which means removing it from utils